Webrew.Client/ClientApp/src/app/services/beer.service.ts

```diff
@@ -1,11 +1,19 @@
 import { Injectable } from '@angular/core';
 import { HttpClient } from '@angular/common/http';
 import { Observable } from 'rxjs';
+import { Beer } from '../shared/models/beer';
+import { Review } from '../shared/models/review';
 
 @Injectable()
 export class BeersService {
-    constructor(private http: HttpClient) { }
+    constructor(private http: HttpClient) {}
     fetchBeers(): Observable<Object> {
```

Comment:
don't forget to give these bad lads a type!

Reply:
updated